bin/bcoin-cli

```
@@ -127,6 +128,14 @@ class CLI {
     this.log(filter);
   }
 
+  async estimateFee() {
+    const blocks = this.config.uint(0, 1);
+
+    const fee = await this.client.estimateFee(blocks);
```

It's likely to never happen, however technically if the status code is 404, this can return `null`, and `fee.rate` would throw a type error, as it would not read the property.

---

Good point - I see similar calls for example return `'Coin not found.'` What should we throw here? `'Fee not available'`?

---

Probably 'Fee not found'

---

Cool, updated.